app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt

```diff
@@ -92,6 +93,15 @@ object NetworkModule : InjectionModule {
                 interceptor = RpcInterceptor(get(), get())
             )
         }
+        single<RpcApi> {
+            getRetrofit(
+                // no need for baseUrl here, we pass URL inside RpcApi
+                baseUrl = "http://localhost/",
```

Comment:
I think that needs to be refactored. 
Move URL passing here. 

@davran1001 fyi